spec.emu

```diff
@@ -34,7 +34,13 @@ markEffects: true
             1. NOTE: This can be used to hide implementation details on the stack trace that aren't useful to user.
           1. Else,
             1. Let _string_ be an implementation-defined string that represents the current stack trace.
-          1. Perform ? SetterThatIgnoresPrototypeProperties(_error_, OrdinaryObjectCreate(*null*), *"stack"*, _string_).
+          1. Perform an implementation-defined choice of either:
```

Comment:
This is an editorial nit, but I would prefer a step more like

> Let \_strategy\_ be an implementation-defined choice of either \~data-property\~ or \~getter\~.

and then typical constructions following that.

Reply:
Also I don't know if you want to permit this choice to be made independently every time `Error.captureStackTrace` is called (as it would with both what you've written and my suggestion) or if you want that choice to be consistent within some unit of execution. For example, it may be a field of the [Agent](https://tc39.es/ecma262/#agent).

Reply:
Not sure how having it a field of the agent change anything, unless that field is restricted from changing. Such restriction could similarly apply to algorithmic steps, no? I thought we had some similar constraints elsewhere.

Reply:
field on agent (for scope) that cannot change works for me

Reply:
Yes, in an agent (or at least in a realm), I don't want the behavior to change between different invocations of `captureStackTrace`.

Reply:
We don't make unobservable constraints like those. I think the widest context we could constrain is an agent cluster.

Reply:
Thanks for the suggestion of using an Agent Record field.